ansible/ncn-master_nodes.yml

```diff
@@ -22,37 +22,42 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-# NCN Master Nodes Play
+# Deprecation Notice: These top level playbooks are replaced by ncn_nodes.yml
+- hosts: all
+  gather_facts: no
+  tasks:
+    - name: DEPRECATION NOTICE
+      debug:
+        msg: "This playbook is marked for deprecation, please use the unified ncn_nodes.yml playbook instead."
+      run_once: true
+      delegate_to: localhost
+
+# NCN Master Nodes Plays, and plays intended for the Management Master Node and associated image
 - hosts: Management_Master
   gather_facts: yes
   any_errors_fatal: true
   remote_user: root
   vars_files:
     - vars/csm_repos.yml
     - vars/csm_packages.yml
-
   roles:
-    # Allow trust of CSM generated keys for elective passwordless SSH;
-    # Enables both passwordless ssh from, and to, all nodes.
     - role: trust-csm-ssh-keys
-    - role: passwordless-ssh
-
-    # Install the platform certificate
     - role: csm.ncn.ca_cert
-
-    # Install CSM repositories and packages
     - role: csm.packages
       vars:
         packages: "{{ ncn_csm_sles_packages }}"
       when: ansible_os_family == "SLE_HPC"
 
-    # Set the root password from the value stored in cray-vault
+# NCN Master Nodes Plays that should never be run in a CFS Image Environment for reasons of security
+- hosts: Management_Master:!cfs_image
+  gather_facts: yes
```

Comment:
Should gather_facts be 'no' here instead? Same question applies for all the master, storage, worker node changes.

Reply:
There are likely additional savings to be had by selectively disabling fact gathering. I was already increasing scope beyond the call in the ticket and can only vouch for the fact gathering:no where indicated.